catalogue/webapp/pages/worksv2.js

```diff
@@ -13,21 +14,50 @@ import StaticWorksContent from '@weco/common/views/components/StaticWorksContent
 import WorkPromo from '@weco/common/views/components/WorkPromo/WorkPromo';
 import Paginator from '@weco/common/views/components/Paginator/Paginator';
 import ErrorPage from '@weco/common/views/components/ErrorPage/ErrorPage';
+import LinkLabels from '@weco/common/views/components/LinkLabels/LinkLabels';
 import {getWorks} from '../services/catalogue/works';
 import {workLink, worksLink} from '../services/catalogue/links';
 
 type Props = {|
   initialQuery: ?string,
   initialWorks: ?CatalogueResultsList | CatalogueApiError,
   initialPage: ?number,
-  filters: Object
+  initialFilters: Object,
+  showCatalogueSearchFilters: boolean
 |}
 
+const workTypes = [
+  { id: 'a', label: 'Books' },
+  { id: 'b', label: 'Manuscripts, Asian' },
+  { id: 'c', label: 'Music' },
+  { id: 'd', label: 'Journals' },
+  { id: 'e', label: 'Maps' },
+  { id: 'f', label: 'E-videos' },
+  { id: 'g', label: 'Videorecordings' },
+  { id: 'h', label: 'Archives and manuscripts' },
+  { id: 'i', label: 'Sound' },
+  { id: 'j', label: 'E-journals' },
+  { id: 'k', label: 'Pictures' },
+  { id: 'l', label: 'Ephemera' },
+  { id: 'm', label: 'CD-Roms' },
+  { id: 'n', label: 'Cinefilm' },
+  { id: 'p', label: 'Mixed materials' },
+  { id: 'q', label: 'Digital images' },
+  { id: 'r', label: '3-D Objects' },
+  { id: 's', label: 'E-sound' },
+  { id: 'u', label: 'Standing order' },
+  { id: 'v', label: 'E-books' },
+  { id: 'w', label: 'Student dissertations' },
+  { id: 'x', label: 'E-manuscripts, Asian' },
+  { id: 'z', label: 'Web sites ' }
```

Comment:
Y NO Y?

Reply:
❓ no idea, there just isn't one - this is info from Sierra.